pkgs/applications/networking/cluster/kuma/default.nix

```diff
@@ -1,19 +1,39 @@
-{ lib, fetchFromGitHub, buildGoModule }:
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, coredns
+, isFull ? true
+, enableGateway ? false
+, pname ? "kuma"
+, version ? "1.3.1"
+, components ? ["kumactl"]
+  ++ lib.optionals isFull [
+    "kuma-cp"
+    "kuma-prometheus-sd"
+    "kuma-dp"
+  ]
+}:
 
-buildGoModule rec {
-  pname = "kumactl";
-  version = "1.3.1";
+buildGoModule {
+  inherit pname version ;
 
   src = fetchFromGitHub {
     owner = "kumahq";
     repo = "kuma";
     rev = version;
+    leaveDotGit = true;
```

Comment:
Why do we need the .git directory?
Version extracting is usually patched because it is reproducible.

Reply:
Many thankś for this, i forgot this in code, i used it to test get version from git, but after some time, i check other packages and see many using same major version in another fields like buildDate and ### gitLastCommit field